packages/web-components/fast-element/docs/guide/next-steps.md

```diff
@@ -7,8 +7,6 @@ custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-comp
 
 We've seen how to use `FASTElement` to declaratively build Web Components. In addition to the basics of element and attribute definition, `FASTElement` also provides a way to declare templates capable of high-performance rendering, and efficient, incremental batched updates. Finally, CSS can easily be associated with an element in a way that leverages core platform optimizations for performance and low memory allocation.
 
-Now that you're familiar with the robust and powerful features of `FASTElement`, you're ready to build your own components and apps. But you don't have to start from scratch there either! If you haven't already explored them, check out our [FAST Components](../components/getting-started), which provide all the basic UI building-blocks you'd expect in a modern component library. You can also leverage the same adaptive design system that our own components use to enable robust theming throughout all you create. Read more on that in [Styling Components](../design/introduction). Finally, you'll want to take advantage of a modern toolset by installing [a powerful editor and plugins](../tools/vscode).
```

Comment:
If someone has a link to the proper Styling Components page I'm happy to update the page. Until then, IMHO it's better to delete the reference, rather than break the build or have a page not found error at run time.

Reply:
I think the equivalent at this point is https://www.fast.design/docs/design-systems/fast-frame#configuring-components. Seems like most of this paragraph gets moved somewhere with the latest packaging and release changes.